### DISCUSSION TITLE
v3.2.2

### DISCUSSION THREAD

**Discussion by osy** [osy86/HaC-Mini#1036] [Releases]
## Changes (v3.2.0)
* Support macOS Ventura
* Add FeatureUnlock kext to support AirPlay mirroring on >= 12.3 (#991) (thanks @goodbest)
* Add AppleIGB kext for LAN2 on >= 12.0 (#947)
* Add timeout for Brcm wifi cards to fix boot issues on >= 12.1 (thanks @goodbest)
* Removed manual UI scale option (OpenCore now detects this automatically)
* Enabled OpenCore support for booting Linux directly

## Changes (v3.2.1)
* Moved AppleIGB to an optional install because it is causing kernel panics
* ~~Secure Boot is now checked by default.~~ This change has been reverted in v3.2.2.

## Changes (v3.2.2)
* Reverted the change in v3.2.1 which set Secure Boot to default. This caused people who did not read the release notes to update with the default settings and inadvertently break their installation. You still need Secure Boot for OTA updates but just like before v3.2.1, you have to manually check the option.
* Fixed an issue where the EFI volume failed to unmount when the installer is run on macOS Ventura due to a change in how `diskutil unmount` worked (#1033)

## Driver Updates (v3.2.0)
* AirportBrcmFixup 2.1.4 -> 2.1.6
* AppleALC 1.7.1 -> 1.7.3
* BrcmPatchRAM 2.6.1 -> 2.6.3
* FeatureUnlock 1.0.8 -> 1.0.9
* Lilu 1.6.0 -> 1.6.1
* NVMeFix 1.0.9 -> 1.1.0
* OpenCore 0.8.0 -> 0.8.2
* Polaris22Fixup v1.3.5 -> v1.3.6
* VirtualSMC 1.2.9 -> 1.3.0
* WhateverGreen 1.5.8 -> 1.6.0

## Driver Updates (v3.2.1)
* AppleALC 1.7.3 -> 1.7.5
* BrcmPatchRAM 2.6.3 -> 2.6.4
* IntelBluetoothFirmware 2.1.0 -> 2.2.0
* Lilu 1.6.1 (OOB) -> 1.6.2
* OpenCore 0.8.3 (OOB) -> 0.8.5
* WhateverGreen 1.6.0 -> 1.6.1

## Known Issues
* OpenIntelWireless does not have a stable build for Ventura and therefore is not included in the package

## Messages
* **Important**: Apple has started requiring Secure Boot for updating to macOS Monterey 12.2 and above (including to macOS Ventura). See the [previous release](https://github.com/osy/HaC-Mini/releases/tag/v3.1.5) for details.
* [As announced a few months ago](https://github.com/osy/HaC-Mini/discussions/578), this project will be transitioning to a long-term support model. macOS Ventura will likely be the last supported version.

<hr /><em>This discussion was created from the release <a href='https://github.com/osy/HaC-Mini/releases/tag/v3.2.2'>v3.2.2</a>.</em>

**Comment by kevinpvo** [comment]
Question for those who have the AppleIGB installed for LAN2 on mac OS 13 -- how stable is this patch for LAN2 for everyone on Ventura? 

The kernel panics mentioned in the 3.2.1 release notes has me debating on if I should install it (because I am using both LAN ports  currently)

Thanks in advance!

**Reply by osy** [reply]
I had a kernel panic maybe once every two weeks which made me disable it.

**Comment by daliang1215** [comment]
how to check the version of hac-mini? I don't know which version I installed?

**Comment by kevinpvo** [comment]
Has anyone updated to the latest version of macOS Ventura, 13.1 yet? 
Currently on 13.0.1 right now and I am just wondering if 13.1 is working okay after update. Thanks!

**Reply by S1999U2** [reply]
I have 13.1 and no complaints
